src/main/deltachat/login.ts

```diff
@@ -115,7 +116,13 @@ export default class DCLoginController extends SplitOut {
 
   async newLogin(credentials: Credentials): Promise<DeltaChatAccount> {
     const newAccountPath = getNewAccountPath()
-    await this.login(newAccountPath, credentials)
+    try {
+      await this.login(newAccountPath, credentials)
+    } catch (error) {
+      log.debug("Detected account creation error, deleting unfinished account", newAccountPath)
```

Comment:
Is this safe that we delete the account on any error? I would prefer to have some string matching here

Reply:
hmm? what kind of string matching?
 > Would be bad if we accidentally delete account folders...

of other accounts you mean?

Reply:
Ahh it only happens if we are doing a new login?

Reply:
yes, see the upper part of the function:
```js
    const newAccountPath = getNewAccountPath()
```